index.js

```diff
@@ -378,7 +378,7 @@
 
 
       bot.chatAddPattern(
-        /^(\[.+?\])? ?([A-Za-z0-9_]{3,16}) left the guild!/,
+        /^Guild > (\[.+?\])? ?([A-Za-z0-9_]{3,16}) left the guild!/,
```

Comment:
not sure if this actually works cuz i dont have access to the message when someone leaves the guild